FlowCrypt/src/androidTest/java/com/flowcrypt/email/ui/activity/EmailManagerActivityTest.kt

```diff
@@ -115,6 +115,7 @@ class EmailManagerActivityTest : BaseEmailListActivityTest() {
 
   @Test
   @DependsOnMailServer
+  @NotReadyForCI
```

Comment:
For these tests marked `NotReadyForCI`, did they use to work? If so, why does this new code make them not work any longer?

Sorry if this is obvious, just wondering!

Reply:
Please look at this comment https://github.com/FlowCrypt/flowcrypt-android/issues/1105#issuecomment-803897034. To be short, sometimes I disable some tests when found that they fail from time to time on CI, BUT work well on my PC. `@NotReadyForCI` was created for that. 

I understand that it is not great, but while we have Node in the app it's the best choice for now. I hope after dropping Node we will have more reliable tests.